README.rst

```diff
@@ -286,6 +286,67 @@ input file. When this setting is missing, the default value ``true`` is used.
 .. |--xunit report.xml| replace:: ``--xunit report.xml``
 .. _`--xunit report.xml`: https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#xunit-compatible-result-file
 
+Parse for Polyspace Failures
+----------------------------
+
+To use the Polyspace checker, a TSV file must be exported from Polyspace.
+You can find instructions on exporting TSV files in the Polyspace documentation.
+
+Exporting the TSV File
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following commands export a TSV file from Polyspace:
+
+.. code-block:: bash
+
+    polyspace-results-export -format csv -results-dir <resultsFolder> <export_options>
+    # or
+    polyspace-results-export -format csv -host <hostName> -run-id <runID> <export_options> <polyspace_access_options>
```

Comment:
Let's include the `-key-mode function-scope` if it proves to be useful

Reply:
Resolved in f9ad210e1a09e4bf230647604f141663d36d7f0b

Reply:
It has proven to be useless for our use case.

Reply:
Reverted in a2724d06e77dac0ad4358f78d96cc1afa6404128